src/app/character-tile/StoreIcon.tsx

```diff
@@ -0,0 +1,43 @@
+import ClassIcon from 'app/dim-ui/ClassIcon';
+import { DimStore } from 'app/inventory/store-types';
+import clsx from 'clsx';
+import React from 'react';
+import styles from './StoreIcon.m.scss';
+
+/**
+ * Show both the store emblem and class icon for a given store.
+ *
+ * Providing a label overrides the class icon.
+ *
+ * @param useBackground uses a portion of the emblem's banner,
+ * which is a little more neutral, instead of the square
+ * version of the emblem
+ */
+export function StoreIcon({
```

Comment:
I think the "emblem + classIcon" and "classIcon over emblem" are actually different enough to have different components, but this is OK for now.

Reply:
sounds like a good idea if i can get them to adequately share code. i'll grab that the next iter